stixcore/ephemeris/manager.py

```diff
@@ -158,8 +158,7 @@ def __init__(self, meta_kernel_path):
 
             self.mod_meta_kernel_path = Path(path)
 
-        # *_, datestamp, version = self.meta_kernel_path.name.split('_')
-        # self.kernel_date = datetime.strptime(datestamp, '%Y%m%d')
+        self.file = self.meta_kernel_path.name
```

Comment:
@nicHoch if we make sure the spice manger  always points to the dated version of the MK this should work at the moment it work but always points to `solo_ANC_soc-flown-mk.tm`  but if we use the other file e.g. `solo_ANC_soc-flown-mk_v107_20210909_001.tm` I think this will work.